client/src/components/loginForm.jsx

```diff
@@ -1,22 +1,56 @@
 import React from 'react';
 
-const LoginForm = () => {
-  return (
-    <form>
-    <div id="username-container">
-      <label htmlFor="email" id="email">
-    Email address
-        <input type="text" name="email" placeholder="Enter username..." />
-      </label>
-    </div>
-    <div id="password-container">
-      <label htmlFor="password" id="password">
-    Password
-        <input type="password" name="password" />
-      </label>
-    </div>
-  </form>
-  )
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      strength: '',
+      color: ''
+    };
+  }
+
+  handleInput = e => {
+    this.setState({
+      [`${e.target.name}`]: e.target.value
+    });
+  };
+
+  render() {
+    return (
+      <form>
+        <div id='username-container'>
+          <label htmlFor='username' id='username'>
+            <span>Username</span>
+            <input
+              onChange={this.handleInput}
+              type='text'
+              name='username'
+              placeholder='Enter username...'
+            />
+          </label>
+        </div>
+        <div id='password-container'>
+          <label htmlFor='password' id='password'>
```

Comment:
What's the tag "htmlFor" used for? Just curious.

Reply:
HTML markup uses a "for" attribute specifies which part of the form a label is bound to. Essentially just provides additional markup for the browser to recognize what to do with the label when it's time to be displayed. 

"htmlFor" is just React's way of naming it as it's own native attribute. Think: "class in HTML vs className in React).